### PR TITLE
Face human detector and landmarks estimators

### PR DESCRIPTION
**Pull request by matemax**

### CODE REVIEW

File: lunavl/sdk/estimators/face_estimators/landmarks.py [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+        asyncEstimate: Literal[True],
+    ) -> AsyncTask[Landmarks68]:
+        ...
+
+    def estimate(
+        self,
+        detection: FaceDetection,
+        landmarksType: Literal[FaceLandmarks.Landmarks5, FaceLandmarks.Landmarks68],
+        asyncEstimate: bool = False,
+    ) -> Union[Landmarks5, Landmarks68, AsyncTask[Landmarks5], AsyncTask[Landmarks68]]:
```
Comment (ddc67cd) [inline lunavl/sdk/estimators/face_estimators/landmarks.py]: Using both overload and unions makes no sense

File: lunavl/sdk/detectors/humandetector.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+
+# alias for detection result
+HumanDetectResult = List[List[HumanDetection]]
+
+
+def collectDetectionsResult(
+    fsdkDetectRes,
+    images: List[Union[VLImage, ImageForDetection]],
+) -> Union[List[List[HumanDetection]]]:
```
Comment (ddc67cd) [inline lunavl/sdk/detectors/humandetector.py]:
```suggestion
) -> HumanDetectResult:
```

File: lunavl/sdk/faceengine/setting_provider.py [hunk 3]
```diff
@@ -1150,6 +1149,137 @@ def landmarks17Threshold(self, value: float) -> None:
...
+
+    @property
+    def faceThreshold(self) -> Optional[float]:
+        """
+        Getter for faceThreshold
+
+        Returns:
+            faceThreshold
+        """
+        return self.getValue("faceThreshold")
```
Comment (ddc67cd) [inline lunavl/sdk/faceengine/setting_provider.py:1176]: Smells like a boilerplate.

File: lunavl/sdk/detectors/humandetector.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+            coreBodyDetection: core body detection
+            image: original image
+            score:  association score of body and face. If face or body is none score must be set to None
+        """
+        self.face = FaceDetection(coreFaceDetection, image) if coreFaceDetection else None
+        self.body = BodyDetection(coreBodyDetection, image) if coreBodyDetection else None
+        self.associationScore = score
+        self._image = image
+
+    def asDict(self) -> Dict[str, Any]:
```
Comment (ddc67cd) [inline lunavl/sdk/detectors/humandetector.py:60]: Not about object behavior, but rather representation. Client's should depend on the public attributes, not the dictionary. And its kinda hard to inherit from a such method.

**Review by ddc67cd — CHANGES_REQUESTED** [review]

File: lunavl/sdk/faceengine/setting_provider.py (hunk 3 above)
Comment (matemax) [reply lunavl/sdk/faceengine/setting_provider.py:1176]: create issue

Comment (matemax) [reply lunavl/sdk/faceengine/setting_provider.py:1176]: https://github.com/matemax/lunasdk/issues/146

File: lunavl/sdk/estimators/face_estimators/landmarks.py (hunk 1 above)
Comment (matemax) [reply lunavl/sdk/estimators/face_estimators/landmarks.py]: remove

**Review by ddc67cd — APPROVED** [review]

**Pull request merged**